### PR TITLE
Fixes 11.12.0 changelog upgrade step & post-update-cmd

### LINKED ISSUE #152: Upgrade steps description -- Address (& others)
This is  a **bug | feature request**.

### Prerequisites
* [ ] Are you running the latest version?
* [x] Are you reporting to the correct repository?
(enso is made of many specialized packages: https://github.com/laravel-enso)
* [x] Did you check the documentation?
* [x] Did you perform a cursory search?

### Description

>To upgrade, manually replace the provided type property where the front-end component is used, replacing the old alias with the mapped class from the addresses.php config. Afterwards, remove the addressable key from the config.

Is pretty unclear; I can't find any changes in the Enso repo that indicate what changed either. Could you provide a code example or reference to a diff on what has to be changed exactly?

Thanks.

### PR DESCRIPTION
**Pull request by jlsjonas**
Fixed #152

### CODE REVIEW

**Comment by jlsjonas** [comment]
2nd commit relates to #153

File: CHANGELOG.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ The upgrade steps are similar to [AddressManager](#morphable)
 - renames the `IsActive` trait into `ActiveState`
 
 ##### Upgrade
-Search in the whole project for `IsActive` and replace it with `ActiveState`.
+Search in the whole project for `IsActive` and replace it with `ActiveState` and `:is-active` with `:active-state`.
```
Comment (aocneanu) [inline CHANGELOG.md]: what's this for?

Comment (aocneanu) [reply CHANGELOG.md]: It's clear now:)

Comment (aocneanu) [reply CHANGELOG.md]: This needs to be removed

**Pull request merged**